src/Features/Core/Portable/Shared/Utilities/SupportedPlatformData.cs

```diff
@@ -31,6 +31,7 @@ public IList<SymbolDisplayPart> ToDisplayParts()
         foreach (var project in projects)
         {
             var text = string.Format(FeaturesResources._0_1, project.Name, Supported(!InvalidProjects.Contains(project.Id)));
+            builder.AddSpace("    ");
```

Comment:
moved the indentation from the resource string to here, so it gets properly tagged as spacing.